src/setup_gmail.py

```diff
@@ -65,7 +65,7 @@ def get_key(self):
             return None
 
     def is_patch(self) -> bool:
-        if re.match(r'\[.+ (\d+)/(\d+)\] .+', self.subject):
+        if re.match(r'\[.+\] .+', self.subject):
```

Comment:
can we perhaps move parse_set_index() into Message to keep the two functions closer together?

```python
   def patch_index(self) -> Tuple[int, int]:
      """Returns the patch index, e.g. (1, 6) for the first patch in a series of 6 patches."""
      if not self.is_patch():
         raise ValueError(...)
      ...
```